CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * Attach zip file of static Braintree xcframework modules to each GitHub release
 * BraintreeCore
   * Fix bug where instantiating multiple feature clients with  universal links in the same view causes `handleOpen` to return in the wrong feature client
+* BraintreePayPal
+  * Add `contactPreference` property to `BTPayPalCheckoutRequest` - This property is required to enable the contact information section in the payment flow.
```

Comment:
We need to pull the latest from main, where is this now was already released so this will be added to the incorrect section of the changelog.